playbook/app/pb_kits/playbook/pb_pill/_pill.tsx

```diff
@@ -11,7 +11,7 @@ type PillProps = {
   data?: {[key: string]: string},
   id?: string,
   text: string,
-  variant?: "success" | "warning" | "error" | "info" | "neutral",
+  variant?: "success" | "warning" | "error" | "info" | "neutral" | "primary",
```

Comment:
Why did we add "primary" as a new variant?

Reply:
It's actually already on the rails version of this kit, don't know why it was missing in the react version. Also, the Label Pill kit has 'primary' as a variant as well and since it's using the Pill kit, typescript was throwing an error since 'variant' on Pill kit and Label Pill kit were typed differently

Reply:
If you look at the playbook website, the Pill example does have primary displayed as a possible variant for this kit
![Screen Shot 2022-08-26 at 12 07 32 PM](https://user-images.githubusercontent.com/73710701/186947244-4340af56-a0ed-43d5-bc25-d0eb935e40d1.png)